conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/LiteralJoinMappingRuleTest.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance.interpreter
+
+import org.apache.spark.sql.AnalysisException
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.{ when => mockWhen }
+import org.scalatest.FunSuite
+
+import za.co.absa.enceladus.conformance.CmdConfig
+import za.co.absa.enceladus.conformance.datasource.DataSource
+import za.co.absa.enceladus.dao.EnceladusDAO
+import za.co.absa.enceladus.model.{ Dataset => ConfDataset }
+import za.co.absa.enceladus.model.MappingTable
+import za.co.absa.enceladus.model.conformanceRule.DropConformanceRule
+import za.co.absa.enceladus.model.conformanceRule.LiteralConformanceRule
+import za.co.absa.enceladus.model.conformanceRule.MappingConformanceRule
+import za.co.absa.enceladus.utils.testUtils.LoggerTestBase
+import za.co.absa.enceladus.utils.testUtils.SparkTestBase
+
+class LiteralJoinMappingRuleTest extends FunSuite with SparkTestBase with LoggerTestBase {
```

Comment:
So were you able to replicate the issue on small scale?

Reply:
Yes this fails locally when removing the config fix